src/components/Cart/User/SignUp.vue

```diff
@@ -61,33 +63,45 @@ export default defineComponent({
     ErrorMessage
   },
   setup() {
+    const store = useStore();
     const router = useRouter();
     const { t } = useI18n();
+    const fbError: Ref<string> = ref("");
     const onSubmit = (values: any) => {
       auth
```

Comment:
Put it into store actions as this is the place for all API calls.
Also use async/await